projects/shared/src/lib/helpers/attribute-filter.helper.ts

```diff
@@ -1,6 +1,6 @@
 import { FilterConditionEnum } from '@tailormap-viewer/api';
 import { AttributeType } from '@tailormap-viewer/api';
-import { FilterConditionModel } from '../models/filter-condition.model';
+import { FilterConditionModel } from '../../../../core/src/lib/filter/models/filter-condition.model';
```

Comment:
This is an illegal dependency on an other lib. The model should probably be moved to shared as well.